### ISSUE TITLE
onTouchedEvent should call setPressed(true) after a delay if the SwipeLayout is in a scroll view

### ISSUE THREAD

**Issue by jyzhang** [daimajia/AndroidSwipeLayout#112]
I'm experiencing this bug when placing SwipeLayout in a recycler view. The pressed states of the background would activate and the item "flashes" when scrolling. This is very annoying. 

I traced the bug to the following code in SwipeLayout#onTouchEvent(...)

```Java
case MotionEvent.ACTION_DOWN:
                mDragHelper.processTouchEvent(event);
                parent.requestDisallowInterceptTouchEvent(true);

                sX = event.getRawX();
                sY = event.getRawY();

                if (touching != null) touching.setPressed(true);
                return true;
```

This doesn't happen in regular views in android because if you look at View#onTouchEvent(...) in [android source](https://android.googlesource.com/platform/frameworks/base/+/refs/heads/master/core/java/android/view/View.java), you'll see:

```Java
case MotionEvent.ACTION_DOWN:
                    mHasPerformedLongPress = false;
                    if (performButtonActionOnTouchDown(event)) {
                        break;
                    }
                    // Walk up the hierarchy to determine if we're inside a scrolling container.
                    boolean isInScrollingContainer = isInScrollingContainer();
                    // For views inside a scrolling container, delay the pressed feedback for
                    // a short period in case this is a scroll.
                    if (isInScrollingContainer) {
                        mPrivateFlags |= PFLAG_PREPRESSED;
                        if (mPendingCheckForTap == null) {
                            mPendingCheckForTap = new CheckForTap();
                        }
                        mPendingCheckForTap.x = event.getX();
                        mPendingCheckForTap.y = event.getY();
                        postDelayed(mPendingCheckForTap, ViewConfiguration.getTapTimeout());
                    } else {
                        // Not inside a scrolling container, so show the feedback right away
                        setPressed(true, x, y);
                        checkForLongClick(0);
                    }
                    break;
```

**Comment by xxxifan** [comment]
Maybe I got the same issue, I'm looking for a workaround.
I used recyclerView inside SwipeRefreshLayout, and SwipeLayout in the RecyclerView, while I swipe the item, if  I scroll down too much, it will trigger both swipe layout and swipe <b>Refresh</b> layout

**Comment by linfaxin** [comment]
fixed at v1.2.0
